jerry-core/ecma/builtin-objects/ecma-builtin-internal-routines-template.inc.h

```diff
@@ -42,6 +42,13 @@
   const ecma_value_t *arguments_list_p, ecma_length_t arguments_list_len
 #define ROUTINE(name, c_function_name, args_number, length_prop_value) \
   static ecma_value_t c_function_name (ROUTINE_ARG_LIST_ ## args_number);
+#define ACCESSOR_VALUE(name, c_getter_func_name, c_setter_func_name, prop_attributes) \
+  static ecma_value_t c_getter_func_name (ROUTINE_ARG_LIST_0); \
+  static ecma_value_t c_setter_func_name (ROUTINE_ARG_LIST_1);
+#define ACCESSOR_RO_VALUE(name, c_getter_func_name, prop_attributes) \
+  static ecma_value_t c_getter_func_name (ROUTINE_ARG_LIST_0);
+#define ACCESSOR_WO_VALUE(name, c_setter_func_name, prop_attributes) \
+  static ecma_value_t c_setter_func_name (ROUTINE_ARG_LIST_1);
```

Comment:
Hm, this might be difficult to adapt for read-only / write-only case. Anyway, just like in the ROUTINE case, you don't need to append a VALUE, since accessor is not a value. We could also clarify these names:

ACCESSOR_READ_WRITE, ACCESSOR_READ_ONLY, ACCESSOR_WRITE_ONLY. Is ACCESSOR_WRITE_ONLY used by the standard? Perhaps we could add it if we need it.